internal/pkg/describe/status_describe.go

```diff
@@ -18,15 +18,19 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/ecs"
 )
 
-const fmtAppRunnerSvcLogGroupName = "/aws/apprunner/%s/%s/service"
+const (
+	fmtAppRunnerSvcLogGroupName = "/aws/apprunner/%s/%s/service"
+	autoscalingAlarmType        = "Auto Scaling"
```

Comment:
What about "Scaling"? Just an idea to make the column less wide 😂

Reply:
I understand your concern! But I don't think "Scaling" is clear enough, unfortunately.